### ISSUE TITLE
Error when using this bundle with native-image 20.1.0

### ISSUE THREAD

**Issue by starkgate** [gradinac/musl-bundle-example#2]
Hello! First, I want to thank you for your prior help with troubleshooting the combining of musl and graalvm.
Now that there is an official 20.1.0 build, I thought I'd give it another try, however I'm getting an error I don't understand when running native-image. It compiles fine without the -H:UseMuslC option. The error log is available here: https://pastebin.com/brvQmqEy

$MUSL_BUNDLE is where I extracted your bundle.

**Comment by gradinac** [comment]
Hey starkgate! No problem at all :)

Thank you for reporting the problem! Issues related to GraalVM and musl should be posted on the official GraalVM page: https://github.com/oracle/graal/issues. This repo is merely a helper I had made to make it easier for people to use the flag.

I have reproduced the error and opened an issue for this: https://github.com/oracle/graal/issues/2353

The problem should be fixed when the PR linked to the issue gets merged.
